packages/postcss-viewport-to-container-toggle/src/core/postcssAdapter.js

```diff
@@ -0,0 +1,94 @@
+const processed = Symbol('processed');
+
+const createPostcssAdapter = (helpers) => {
```

Comment:
I feel this file is an adapter, while the other adapter is just the tool itself no ?